app/src/main/java/org/schabi/newpipe/MainActivity.java

```diff
@@ -603,6 +603,7 @@ public void onBackPressed() {
     public void onRequestPermissionsResult(final int requestCode,
```

Comment:
Shouldn't these also be migrated to the new way of handling permission results introduced in `1.3.0`? (also `RouterActivity.java`)

Reply:
Yeah, but I'm going to do it in another PR, as mentioned above :)

Reply:
Oh, yes, sorry ;-)